pvacseq/lib/pipeline.py

```diff
@@ -297,6 +303,17 @@ def __init__(self, **kwargs):
         Pipeline.__init__(self, **kwargs)
         self.peptide_sequence_length = 31
 
+    def generate_fasta(self):
+        print("Generating Variant Peptide FASTA File")
+        sys.stdout.flush()
+        lib.generate_fasta.main([
+            self.tsv_file_path(),
+            str(self.peptide_sequence_length),
+            '9',
```

Comment:
I found the rationale for the `9` in #168, but it might be nice to document its significance in some way.